docs/frontend/testing/frontend.md

```diff
@@ -6,45 +6,6 @@ sidebar_position: 1
 
 ## Table of Contents
 
-<!-- toc -->
-
-* [Design + Engineering Process for new components](#design--engineering-process-for-new-components)
-  * [Design delivers component design](#design-delivers-component-design)
-  * [Engineering](#engineering)
-* [Testing](#testing)
-  * [Writing Tests](#writing-tests)
-  * [Unit Test Runners and Libraries](#unit-test-runners-and-libraries)
-  * [Browser Testing](#browser-testing)
-  * [Storybook Testing](#storybook-testing)
-* [Code Style](#code-style)
-  * [Auto-formatting](#auto-formatting)
-  * [Linting](#linting)
-  * [File Layout & Naming](#file-layout--naming)
-  * [Presentation vs. Container components](#presentation-vs-container-components)
-  * [Function Declarations](#function-declarations)
-  * [Ordering imports](#ordering-imports)
-  * [Using Redux](#using-redux)
-  * [Creating Forms](#creating-forms)
-  * [CSS Styling Standards](#css-styling-standards)
-    * [Using Sass and CSS Modules](#using-sass-and-css-modules)
-    * [Classnames](#classnames)
-    * [rem vs. em](#rem-vs-em)
-    * [BEM](#bem)
-    * [USWDS](#uswds)
-* [Tooling](#tooling)
-  * [Sublime Plugins](#sublime-plugins)
-  * [WebStorm](#webstorm)
-  * [VS Code](#vs-code)
-  * [vi](#vi)
-  * [Browser Extensions](#browser-extensions)
-* [Learning](#learning)
-  * [JavaScript Concepts](#javascript-concepts)
-  * [Resources](#resources)
-
-Regenerate with "pre-commit run -a markdown-toc"
-
-<!-- tocstop -->
-
```

Comment:
Sorry for this deletion being included in this PR, but TOC don't need to be generated anymore because of Docusaurus. This page now includes a TOC on the right-hand side for Desktop.

<img width="1616" alt="Captura de Pantalla 2021-12-07 a la(s) 3 32 38 p m" src="https://user-images.githubusercontent.com/706004/145102017-063f66db-ff44-4b70-a077-6db711fb2963.png">